src/pystage/core/_sensing.py

```diff
@@ -199,7 +199,21 @@ def __init__(self):
         super().__init__()
 
     def sensing_touchingobject_pointer(self):
-        pass
```

Comment:
This solution is not general enough. We cannot just look at a rectangle or circle sprite. It could be any sprite with any outline. Try this scratch project written as text here:

while green flag clicked:
    forever:
        say ( touching ( mouse ) ) 

You can move your pointer around the screen and get very near the cat, like in the armpit, between the ears, between the legs, etc., and the value is false. It is only true when the mouse pointer is over the actual pixels of the cat sprite. So, we'll need a much much more general solution than this.

Reply:
I think it doesn't, the PR is just not named correctly. The mask is created from non-transparent pixels. But for performance reasons, there should still be a rectangular collision test first and only of it succeeds, we check for pixel perfect collision.